geometry_msgs/Pose p_i
geometry_msgs/Pose p_f
---
trajectory_msgs/JointTrajectory traj

